cmd/peer.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/textileio/textile-go/core"
+	"gopkg.in/abiosoft/ishell.v2"
+)
+
+func init() {
```

Comment:
This is the basic idea... this thing conforms to a cmd tool interface AND a shell interface, so we get both